app/views/podcasts/show.rss.builder

```diff
@@ -106,6 +106,10 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
       xml.podcast :guid, @podcast.guid
     end
 
+    if @podcast.subscribe_links.present?
```

Comment:
placeholder material in preparation for the json PR. but this will be where the RSS gets updated after the json of links is uploaded.